libexec/cli/action_argparser.sh

```diff
@@ -116,22 +116,24 @@ while true; do
         ;;
         -n|--nv)
             shift
-            for i in `ldconfig -p | grep -E "/libnv|/libcuda|/libEGL|/libGL|/libnvcu|/libvdpau|/libOpenCL|/libOpenGL"`; do
-                if [ -f "$i" ]; then
-                    message 2 "Found NV library: $i\n"
-                    if [ -z "${SINGULARITY_CONTAINLIBS:-}" ]; then
-                        SINGULARITY_CONTAINLIBS="$i"
-                    else
-                        SINGULARITY_CONTAINLIBS="$SINGULARITY_CONTAINLIBS,$i"
+            for lib in $(cat $SINGULARITY_sysconfdir"/singularity/nvliblist.conf" | grep -v "^#"); do
+                for i in $(ldconfig -p | grep "${lib}"); do
```

Comment:
Ah.  Thanks for the clarification.  I see what you mean.  I just implemented it.  Will fire off another pull request to @mrmhodak in a minute.